src/plugins/guided_onboarding/public/components/quit_guide_modal.tsx

```diff
@@ -59,7 +59,7 @@ export const QuitGuideModal = ({
           <p>
             {i18n.translate('guidedOnboarding.quitGuideModal.modalDescription', {
               defaultMessage:
-                'You can restart anytime by opening the Setup guide from the Help menu.',
+                'You can restart anytime by opening the Setup guides link from the Help menu.',
```

Comment:
Not sure if this one should be plural or if the word link is needed.  @kellyemurphy?